Kudu.Core/Deployment/WellKnownEnvironmentVariables.cs

```diff
@@ -36,5 +36,6 @@ internal static class WellKnownEnvironmentVariables
         public const string SiteBitness = "SITE_BITNESS";
 
         public const string GoWebConfigTemplate = "GO_WEB_CONFIG_TEMPLATE";
+        public const string SelectLatestVersionCommandKey = "SELECT_LATEST_VERSION_CMD";
```

Comment:
I think the reasoning is that the SelectLatest command can in theory be used for any such scenario, so it's not Go specific, even if as it stands that's the use case. So I'm ok with the naming, but a few more comments in the code about what this command is would be helpful.